database_admin/schema/create_schema.sql

```diff
@@ -741,6 +741,7 @@ CREATE TABLE IF NOT EXISTS advisory_metadata
     severity_id      INT,
     package_data     JSONB,
     cve_list         JSONB,
+    reboot_required  BOOLEAN NOT NULL DEFAULT true,
```

Comment:
Isn't false more appropriate default? I.e. reboot is not required unless stated explicitly?

Reply:
Ok, updated.